WMIAdventure/frontend/src/js/components/battle/organisms/BattleView/BattleView.js

```diff
@@ -32,23 +40,76 @@ class BattleView extends React.Component {
         // states for items initial animation movement
         enemyCompactCardTranslateX: '-100vw',
         userCompactCardTranslateX: '100vw',
+
         enemyMiniCardsTranslateX: ['-100vw', '-100vw', '-100vw', '-100vw', '-100vw'],
         userMiniCardsTranslateX: ['100vw', '100vw', '100vw', '100vw', '100vw'],
+
         enemyStateContainerTranslateX: '-100vw',
         userStateContainerTranslateX: '100vw',
 
+        enemyCompactCardTranslateY: '0',
+        userCompactCardTranslateY: '0',
+
         // states for hp and shield points
         enemyHp: '0',
         userHp: '0',
         enemyShield: '0',
         userShield: '0',
 
-        // prototype data
-        cardLevels : [3, 3, 2, 1, 1],
-        icons : [icon1, icon2, icon3, icon4, icon5],
         // states to handle cards orders, pass to CompactCardView and MiniCardView as props
-        cardsUserOrder : [1, 2, 3, 4, 5], // this means: first card on first place and so on
         cardsEnemyOrder : [1, 2, 3, 4, 5],
+        cardsUserOrder : [1, 2, 3, 4, 5], // this means: first card on first place and so on
+
+        // full cards action
+        enemyFullCardAction: {
+            visible: false,
+            opacity: '0',
+            translateY: '-100vh',
+        },
+        userFullCardAction: {
+            visible: false,
+            opacity: '0',
+            translateY: '100vh',
+        },

```

Comment:
Nie rozumiem czemu `enemy` i `user` są osobnymi obiektami. Jaki jest use case gdzie potrzebujemy na raz kartę usera i enemy? Wystarczy na to jeden stan i np przechowywać albo w nim albo osobno informacje o tym czy to karta usera czy enemy.

Reply:
Tak łatwej było, bo inne translateY są na początku.
Ale mogę to zrobić w jednym, po prostu przestawię translateY w odpowiednim momencie kiedy użytkownik nie będzie widział.